dashboard/app/controllers/ai_diff_controller.rb

```diff
@@ -0,0 +1,66 @@
+class AiDiffController < ApplicationController
+  include AiDiffBedrockHelper
+  # authorize_resource class: false
+  before_action :require_levelbuilder_mode_or_test_env, except: [:chat_completion]
+  load_and_authorize_resource except: [:chat_completion]
+
+  # params are
+  # inputText:
+  # lessonId:
+  # unitDisplayName:
+  # sessionId:
+  # POST /ai_diff/chat_completion
+  def chat_completion
+    return render status: :forbidden, json: {} unless current_user&.teacher?
+    is_ai_diff_experiment_enabled = current_user && SingleUserExperiment.enabled?(user: current_user, experiment_name: 'ai-differentiation')
+    return render status: :forbidden, json: {} unless is_ai_diff_experiment_enabled
+    unless has_required_params?
+      return render status: :bad_request, json: {}
+    end
+
+    @session_id = params[:sessionId].presence ? params[:sessionId] : nil
```

Comment:
Per the [documentation](https://apidock.com/rails/Object/presence) for `presence` it will return either the object (if present) or `nil` otherwise, so no need to repeat that logic here.

```suggestion
    @session_id = params[:sessionId].presence
```